src/azure/cli/commands/__init__.py

```diff
@@ -89,7 +89,6 @@ def _delay(self):
         time.sleep(self.poll_interval_ms / 1000.0)
 
     def __call__(self, poller):
-        logger.warning(self.start_msg)
```

Comment:
i like this change personally, but for the people get used to interactive background, is it possible we can still output the starting message, but just don't prefix with "warning:"?